packages/data-provider/src/config/config.ts

```diff
@@ -39,12 +43,14 @@ export type AppConfig = {
 	logger: LoggerConfig;
 	schemaService: SchemaServiceConfig;
 	idService: IdServiceConfig;
+	audit: AuditConfig;
 };
 
 /**
  * Base Dependencies required for utils/services
  */
 export interface BaseDependencies {
+	audit: AuditConfig;
```

Comment:
This is in the correct place alphabetically but the location of `idService` is undoing your hard work ;P

Reply:
sorted alphabetically.